disnake/automod.py

```diff
@@ -127,11 +135,21 @@ class AutoModBlockMessageAction(AutoModAction):
 
     _metadata: AutoModBlockMessageActionMetadata
 
-    def __init__(self) -> None:
+    def __init__(self, custom_message: str) -> None:
```

Comment:
this should have a default of None

Reply:
717deddb068512229a24b634bab8bbb729ce00ac